Mathlib/Algebra/Order/GroupWithZero/Unbundled.lean

```diff
@@ -1113,6 +1113,13 @@ end CancelMonoidWithZero
 section GroupWithZero
 variable [GroupWithZero G₀]
 
+section Preorder
+variable [Preorder G₀] [ZeroLEOneClass G₀]
+
+lemma div_self_le_one (a : G₀) : a / a ≤ 1 := by obtain rfl | ha := eq_or_ne a 0 <;> simp [*]
```

Comment:
```suggestion
/-- See `div_self` for the version with equality in `Group G`. -/
lemma div_self_le_one (a : G₀) : a / a ≤ 1 := by obtain rfl | ha := eq_or_ne a 0 <;> simp [*]
```
I think something like this would be helpful for autocomplete users (although I guessed both the name and the correct generality for the other version, feel free to amend that)